okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/ConcurrencyLimiters.java

```diff
@@ -209,12 +212,32 @@ public void onDropped() {}
 
         private final Key limiterKey;
         private final Supplier<SimpleLimiter<Void>> limiterFactory;
-        private final LeakDetector<Limiter.Listener> leakDetector = new LeakDetector<>(Limiter.Listener.class);
+        private final LeakDetector<Limiter.Listener> leakDetector;
 
         DefaultConcurrencyLimiter(Key limiterKey, Supplier<SimpleLimiter<Void>> limiterFactory) {
             this.limiterKey = limiterKey;
             this.limiterFactory = limiterFactory;
             this.limiter = limiterFactory.get();
+            this.leakDetector = new LeakDetector<>(
+                    Limiter.Listener.class,
+                    // Create stack traces for five minutes after a leak is detected
```

Comment:
also worth updating the log lines in the LeakDetector class as this is now unnecessary?

> To get a stack trace for the call where the acquire happened, set log level to TRACE.